webview.h

```diff
@@ -144,17 +145,19 @@ WEBVIEW_API void webview_return(webview_t w, const char *seq, int status,
 namespace webview {
 using dispatch_fn_t = std::function<void()>;
 
-inline std::string url_encode(const std::string &s) {
+// Percent-encodes (%xx) each character of a string.
+inline std::string percent_encode(const std::string &s) {
```

Comment:
I just noticed that `url_encode` isn't static and is accessible by users with `webview::url_encode`). I don't see any bindings that use it, and `url_decode` was static. We *might* be able to make `url_encode` static as well, unless there was or is a reason otherwise.

Reply:
In my opinion it would be good to consider moving everything not part of the public API into a nested namespace to remove doubts in the future.

Reply:
I opened the following PR to open a discussion regarding the accessibility of functions that are not intended to be public and to work on a solution.

  * #748